thirdeye/thirdeye-frontend/app/pods/manage/alert/template.hbs

```diff
@@ -130,84 +160,188 @@
+              <tbody>
+                {{#each paginatedFilteredAnomalies as |anomaly|}}
+                  <tr class="te-alert-page-anomalies__table-row te-alert-page-anomalies__table--body">
+                     <td class="te-alert-page-anomalies__table--cell">
+                      <ul class="te-alert-page-anomalies__table-list">
+                        <li class="te-alert-page-anomalies__table-list--stronger">{{anomaly.startDateStr}}</li>
+                        <li class="te-alert-page-anomalies__table-list--lighter">{{anomaly.durationStr}}</li>
+                      </ul>
+                     </td>
+                     <td class="te-alert-page-anomalies__table--cell">
+                      <ul class="te-alert-page-anomalies__table-list">
+                       {{#each anomaly.dimensionList as |dimension|}}
+                          <li class="te-alert-page-anomalies__table-list--smaller">
+                            {{dimension.dimensionKey}}: {{dimension.dimensionVal}}
+                          </li>
+                       {{else}}
+                          -
+                       {{/each}}
+                      </ul>
+                     </td>
+                     <td class="te-alert-page-anomalies__table--cell">{{anomaly.severityScore}}</td>
+                     <td class="te-alert-page-anomalies__table--cell">
+                      <ul class="te-alert-page-anomalies__table-list">
+                        <li>{{anomaly.shownCurrent}} / {{anomaly.shownBaseline}}</li>
+                        <li class="te-alert-page-anomalies__table-list--warning">({{anomaly.shownChangeRate}}%)</li>
+                      </ul>
+                     </td>
+                     <td class="te-alert-page-anomalies__table--cell">
+                        {{#if anomaly.showResponseSaved}}
+                          <i class="te-alert-page-anomalies__glyph--status glyphicon glyphicon-ok-circle"></i>
+                        {{/if}}
+
+                        {{#if anomaly.showResponseFailed}}
+                          <i class="te-alert-page-anomalies__glyph--status glyphicon glyphicon-remove-circle"></i>
+                        {{/if}}
+
+                        {{#power-select
+                          triggerId=anomaly.anomalyId
+                          triggerClass="te-alert-page-anomalies__table-select"
+                          options=responseOptions
+                          searchEnabled=false
+                          selected=anomaly.anomalyFeedback
+                          onchange=(action "onChangeAnomalyResponse" anomaly)
+                          as |response|
+                        }}
+                          {{response}}
+                        {{/power-select}}
+
+                        <a target="_blank" class="te-alert-page-anomalies__table-link" href="http://thirdeye.corp.linkedin.com/thirdeye#anomalies?anomaliesSearchMode=id&pageNumber=1&anomalyIds={{anomaly.anomalyId}}">investigate</a>
```

Comment:
please remove the `http://thirdeye.corp.linkedin.com` (it's linkedin specific data). 
a relative path is fine (on prod, /app and /thirdeye are on the same level)